numba/core/dispatcher.py

```diff
@@ -983,6 +983,12 @@ def compile(self, sig):
             flags = self.flags
             args, return_type = sigutils.normalize_signature(sig)
 
+            # for objmode we should have ffi_forced_objects as args
+            # also there is probably no benefit in rewrites passes, so turn them off
```

Comment:
```suggestion
            # also there are no applicable rewrite passes, so turn them off
```
there's no rewrite pass triggered in the `objmode` pipeline
https://github.com/numba/numba/blob/58cd2ee50950bed13b129a07c83a7606340064fc/numba/core/compiler.py#L560-L588